ambari_plugin/common-services/DOLPHIN/1.3.0/package/scripts/dolphin_api_service.py

```diff
@@ -45,10 +45,6 @@ def start(self, env):
         init_cmd=format("sh " + params.dolphin_home + "/script/create-dolphinscheduler.sh")
         Execute(init_cmd, user=params.dolphin_user)
 
-        #upgrade
-        upgrade_cmd=format("sh " + params.dolphin_home + "/script/upgrade-dolphinscheduler.sh")
```

Comment:
I think the problem is in the schema file. Not here.
In order to ensure that the table structure metadata is up to date, we need to execute the upgrade-dolphinscheduler.sh script at startup. This script will automatically check the current version number and execute the corresponding upgrade sql file.

Therefore, we require that the sql file in the schema must meet the ability of repeatable execution.
In order to achieve this goal, if we need to create the table t_ds_process_definition_version, then we should use `create table IF NOT EXISTS t_ds_process_definition_version`

Reply:
+1